Berksfile

```diff
@@ -3,5 +3,8 @@ source 'https://supermarket.chef.io'
 
 metadata
 
-cookbook 'apt'
-cookbook 'yum'
+group :integration do
+  cookbook 'apt'
+  cookbook 'yum'
```

Comment:
Yes, that's because these particular images need an `apt-get update` run, which simply downloads the package lists from the repositories and "updates" them to get information on the newest versions of packages and their dependencies.

Here's the modified `platform` section with `- RUN /usr/bin/apt-get update` - it's similar to the same section for the `inspec` project.  Once I changed this for ubuntu 12, 14, 15 and debian8 they all converged fine.

```
platforms:
- name: ubuntu-12.04
  driver:
    image: ubuntu:12.04
    intermediate_instructions:
    - RUN /usr/bin/apt-get update
- name: ubuntu-14.04
  driver:
    image: ubuntu:14.04
    intermediate_instructions:
    - RUN /usr/bin/apt-get update
- name: ubuntu-15.10
  driver:
    image: ubuntu:15.10
    intermediate_instructions:
    - RUN /usr/bin/apt-get update
    pid_one_command: /bin/systemd
- name: centos-6.6
  driver:
    image: centos:6.6
- name: centos-6.7
  driver:
    image: centos:6.7
    intermediate_instructions:
      - RUN yum install -y initscripts
- name: centos-7
  driver:
    image: centos:7
    pid_one_command: /usr/lib/systemd/systemd
- name: oracle-6.6
  driver:
    image: oraclelinux:6.6
- name: oracle-6.7
  driver:
    image: oraclelinux:6.7
- name: oracle-7.1
  driver:
    image: oraclelinux:7.1
    pid_one_command: /usr/lib/systemd/systemd
- name: debian-7
  driver:
    image: debian:7
    intermediate_instructions:
    - RUN /usr/bin/apt-get update
    - RUN /usr/bin/apt-get install -y procps
- name: debian-8
  driver:
    image: debian:8
    intermediate_instructions:
    - RUN /usr/bin/apt-get update
    pid_one_command: /bin/systemd
```